CMake/unix_config.cmake

```diff
@@ -47,6 +47,61 @@ macro(os_set_flags)
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
     endif()
     
+    ###############
+    # According to SDLE we need to add the following flags for additional security:
```

Comment:
Let's condition the flags according to the specific compiler mentioned in the doc